recipes/mageck/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: mageck
-  version: "0.5.4"
+  version: "0.5.5"
 
 source:
   git_url: https://bitbucket.org/liulab/mageck.git
-  git_rev: v0.5.4
+  git_rev: v0.5.5
```

Comment:
What I meant is, if you can use `url: https://bitbucket.org/liulab/mageck/get/v0.5.5.tar.gz` and `md5:  f55421c0a32907592dcf84643e60139c` instead of git_url